src/overview.stories.mdx

```diff
@@ -0,0 +1,433 @@
+import { Meta } from '@storybook/addon-docs';
+import {
+  Grid,
+  Button,
+  Checkbox,
+  Input,
+  MultiValueInput,
+  Password,
+  Radio,
+  Range,
+  SearchBarV2,
+  SegmentedToggle,
+  SegmentedToggleItem,
+  Select,
+  Stack,
+  Heading,
+  SelectableGroup,
+  Switch,
+  TextArea,
+  SemanticModal,
+  Badge,
+  Banner,
+  Breadcrumbs,
+  BreadcrumbItem,
+  ButtonGroup,
+  Callout,
+  CloseButton,
+  Collapsible,
+  DateRangePicker,
+  FileSelector,
+  HexGrade,
+  Inline,
+  Icon,
+  IconButton,
+  Nav,
+  NavItem,
+  Pill,
+  Signal,
+  SingleDatePicker,
+  SortableList,
+  Spinner,
+  Stepper,
+  Step,
+  Label,
+  Toast,
+  Tooltip,
+  Paragraph,
+  UserAvatar,
+  Pagination,
+  Drawer,
+} from './components';
+import { DSProvider, SpaceSizes } from './theme';
+import styled from 'styled-components';
+import { getColor, getRadii, getSpace } from './utils';
+import {
+  HeadingSizes,
+  HeadingVariants,
+} from './components/typographyLegacy/Heading/Heading.enums';
+import { useState } from 'react';
+import { ButtonColors } from './components/Button/Button.enums';
+import { noop, sign } from 'ramda-adjunct';
+import { BadgeSizes } from './components/Badge/Badge.enums';
+import { BrowserRouter } from 'react-router-dom';
+import { themes } from '@storybook/theming';
+
+<Meta title="Overview" />
+
+export const StickerContainer = styled.a`
```

Comment:
@julesbarr this being an `a` tag causes I'm being immediately redirected to some other page. If I change this to `div` it works fine.

Reply:
I would probably add a button on the bottom of the card with the text "See documentation". It will be also better because users cannot interact with component in the cards if the whole card acts as a button